newton/examples/example_mjwarp_tendon.py

```
@@ -0,0 +1,239 @@
+        
+        # Save renderer output
+        if self.renderer and hasattr(self.renderer, 'save'):
+            self.renderer.save()
+            print(f"\nAnimation saved to: {self.renderer.stage_path}")
+    
+    def __del__(self):
+        # Clean up temporary directory
+        if hasattr(self, 'tmpdir') and self.tmpdir:
+            try:
+                self.tmpdir.cleanup()
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)